README.md

```diff
@@ -22,20 +22,31 @@
 Windows is not an explicitly supported environment, although where possible Palisade has been made compatible.  
 For Windows developer environments, we recommend setting up [WSL](https://docs.microsoft.com/en-us/windows/wsl/).
 
+## Getting started
+For an overview of Palisade, start with the Palisade introduction and the accompanying guides: QuickStart Guide; and Developer Guide: [Palisade README](https://github.com/gchq/Palisade).
```

Comment:
Direct links to both of these?

Reply:
I think it is better with just the one link, the others are just examples of the types of documents you can find there

Reply:
left is a is.  This it make more sense this way.  Point them to the starting point